addon/components/rdf-input-fields/currency-input.hbs

```diff
@@ -0,0 +1,43 @@
+{{#unless @inTable}}
+  <AuLabel
+    @error={{this.hasErrors}}
+    @warning={{this.hasWarnings}}
+    @required={{this.isRequired}}
+    for={{this.inputId}}
+  >
+    {{@field.label}}
+  </AuLabel>
+{{/unless}}
+
+<AuButtonGroup @inline={{true}}>
+  <AuButton
+    @skin="primary"
+    @disabled={{true}}
+  >{{this.currencySymbol}}</AuButton>
```

Comment:
Creative solution, but this reminds me that we need to add an ["input group" feature](https://github.com/appuniversum/ember-appuniversum/issues/368) to Appuniversum 😄, or some way to display custom content instead of only the icons we support now.

I noticed that the cursor will show a 🚫 icon when hovering the euro sign, because it's a disabled button. I wonder if we should add some style overrides to the addon for this specific case, until Appuniversum supports input groups? (`pointer-events: none` most likely?)

Reply:
Unfortunately this didn't work with the pointer events none.. Also tried the `cursor: pointer !important` but it did not work..

As t is a new component i create a class for the currencyInput component. Maybe see if we want this custom css or just the disabled cursor event

Reply:
![image](https://github.com/lblod/ember-submission-form-fields/assets/36266973/a8b9aa42-b990-4ebf-9d66-dbaad7138d9b)

Reply:
> Unfortunately this didn't work with the pointer events none.. Also tried the cursor: pointer !important but it did not work..

Strange, I tried that in my dev tools 😄 maybe it works in Firefox. Anyway, custom styles is nice as well 👍.